### PR TITLE
Refactor abi binding files

### PR DESCRIPTION
**Pull request by gwen917**
## Why this should be merged
- Rename `abis` to `abi-bindings`
- Match the structure in `src`

## How this was tested
- CI

### CODE REVIEW

File: scripts/abi_go_bindings.sh [hunk 1]
```diff
@@ -57,21 +57,23 @@ if [[ -z "${CONTRACT_LIST}" ]]; then
...
+    dir=$(dirname $path)
     abi_file=$TELEPORTER_PATH/contracts/out/$contract_name.sol/$contract_name.abi.json
     if ! [ -f $abi_file ]; then
         echo "Error: Contract $contract_name abi file not found"
         exit 1
     fi
 
     echo "Generating Go bindings for $contract_name..."
-    mkdir -p $TELEPORTER_PATH/abis-bings
+    mkdir -p $TELEPORTER_PATH/abis-bings/$dir/$contract_name
```
Comment (geoff-vball) [inline scripts/abi_go_bindings.sh:72]: The PR said we were using `abi-bindings` but this is `abis-bings`

Comment (gwen917) [reply scripts/abi_go_bindings.sh]: fixed

**Review by geoff-vball — APPROVED** [review]
LGTM 👍

**Review by cam-schultz — DISMISSED** [review]
LGTM 👍

**Review by minghinmatthewlam — APPROVED** [review]
LGTM

**Review by cam-schultz — APPROVED** [review]

**Comment by geoff-vball** [comment]
I think you'll have to fix up `basic_one_way_send.go` to get the tests to pass

**Review by geoff-vball — APPROVED** [review]
👍

**Review by cam-schultz — APPROVED** [review]

**Pull request merged**